charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.52.0
-version: 25.21.0
+version: 25.21.1
```

Comment:
```suggestion
version: 25.22.0
```

Reply:
In my opinion, the change is indeed a documentation patch.